---
layout: page
title: Ilya Vorontsov
permalink: /about/
---

Hi, I am Ilya Vorontsov (Илья Воронцов), researcher and developer from Moscow, Russia.

I study gene regulation in [Vavilov Institute of General Genetics](http://vigg.ru/) using bioinformatician data analysis (since 2012).

My [Google scholar profile](https://scholar.google.ru/citations?user=hRn71foAAAAJ).
<details>
  <style>
    .authors{ font-style: italic; }
    .title{ font-weight: bolder; }
    .publisher{  }
  </style>
  <summary>Selected articles</summary>
  <ol>
  <li>(2019) <span class="title">What do neighbors tell about you: the local context of cis-regulatory modules complicates prediction of regulatory variants.</span> <span class="authors">Dmitry D Penzar, Arsenii O Zinkevich, Ilya E Vorontsov, Vasily V Sitnik, Alexander V Favorov, Vsevolod J Makeev, Ivan V Kulakovskiy.</span><br/><span class="publisher">Frontiers in genetics, 31 October 2019. <a href="https://www.frontiersin.org/articles/10.3389/fgene.2019.01078/full">doi:10.3389/fgene.2019.01078</a></span></li>

  <li>(2017) <span class="title">HOCOMOCO: Towards a complete collection of transcription factor binding models for human and mouse via large-scale ChIP-Seq analysis.</span> <span class="authors">Ivan V Kulakovskiy, Ilya E Vorontsov, Ivan S Yevshin, Ruslan N Sharipov, Alla D Fedorova, Eugene I Rumynskiy, Yulia A Medvedeva, Arturo Magana-Mora, Vladimir B Bajic, Dmitry A Papatsenko, Fedor A Kolpakov, Vsevolod J Makeev.</span><br/><span class="publisher">Nucleic Acids Research, Volume 46, Issue D1, 4 January 2018, Pages D252–D259. <a href="https://academic.oup.com/nar/article/46/D1/D252/4616875">doi:10.1093/nar/gkx1106</a></span></li>

  <li>(2016) <span class="title">Negative selection maintains transcription factor binding motifs in human cancer.</span> <span class="authors">Ilya E Vorontsov, Grigory Khimulya, Elena N Lukianova, Daria D Nikolaeva, Irina A Eliseeva, Ivan V Kulakovskiy and Vsevolod J Makeev.</span><br/><span class="publisher">BMC Genomics 2016, 17(Suppl 2):395. <a href="http://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2728-9">doi:10.1186/s12864-016-2728-9</a></span></li>

  <li>(2016) <span class="title">HOCOMOCO: expansion and enhancement of the collection of transcription factor binding sites models.</span> <span class="authors"> Ivan V. Kulakovskiy , Ilya E. Vorontsov , Ivan S. Yevshin, Anastasiia V. Soboleva, Artem S. Kasianov, Haitham Ashoor, Wail Ba-alawi, Vladimir B. Bajic, Yulia A. Medvedeva, Fedor A. Kolpakov and Vsevolod J. Makeev.</span><br/><span class="publisher">Nucleic acids research (2015). <a href="http://nar.oxfordjournals.org/content/44/D1/D116.full">doi:10.1093/nar/gkv1249</a></span></li>

  <li>(2015) <span class="title">PERFECTOS-APE - Predicting Regulatory Functional Effect of SNPs by Approximate P-value Estimation.</span> <span class="authors">I.E. Vorontsov, I.V. Kulakovskiy, G.Khimulya, D.D. Nikolaeva and V.J. Makeev.</span><br/><span class="publisher">Proceedings of the International Conference on Bioinformatics Models, Methods and Algorithms (BIOINFORMATICS-2015), pages 102-108.<a href="http://www.scitepress.org/DigitalLibrary/PublicationsDetail.aspx?ID=dAhqokIeyfg=&t=1">doi:10.5220/0005189301020108</a></span></li>

  <li>(2015) <span class="title">EpiFactors: a comprehensive database of human epigenetic factors and complexes.</span> <span class="authors">Yulia A Medvedeva, Andreas Lennartsson, Rezvan Ehsani, Ivan V Kulakovskiy, Ilya E Vorontsov, Pouda Panahandeh, Grigory Khimulya, Takeya Kasukawa, Finn Drabløs, FANTOM Consortium.</span><br/><span class="publisher">Database (2015). <a href="http://database.oxfordjournals.org/content/2015/bav067.full">doi:10.1093/database/bav067</a></span></li>

  <li>(2014) <span class="title">A promoter-level mammalian expression atlas.</span> <span class="authors">The FANTOM Consortium.</span><br/><span class="publisher">Nature 507, 462–470 (27 March 2014). <a href="https://www.nature.com/articles/nature13182">doi:10.1038/nature13182</a></span></li>

  <li>(2013) <span class="title">In silico motif analysis suggests an interplay of transcriptional and translational control in mTOR response.</span> <span class="authors">Eliseeva IA, Vorontsov IE, Babeyev KE, Buyanova SM, Sysoeva MA, Kondrashov FA, Kulakovskiy IV.</span><br/><span class="publisher">Translation 2013; 1:e27469; <a href="http://www.tandfonline.com/doi/abs/10.4161/trla.27469">doi:10.4161/trla.27469</a></span></li>

  <li>(2013) <span class="title">Jaccard index based similarity measure to compare transcription factor binding site models.</span> <span class="authors">Ilya E Vorontsov, Ivan V Kulakovskiy and Vsevolod J Makeev.</span><br/><span class="publisher">Algorithms for Molecular Biology, 2013, 8:23; <a href="http://almob.biomedcentral.com/articles/10.1186/1748-7188-8-23">doi:10.1186/1748-7188-8-23</a></span></li>

  <li>(2013) <span class="title">From binding motifs in ChIP-Seq data to improved models of transcription factor binding sites.</span> <span class="authors">I. Kulakovskiy, V. Levitsky, D. Oshchepkov, L. Bryzgalov, I. Vorontsov, V. Makeev.</span><br/><span class="publisher">Journal of Bioinformatics and Computational Biology (J Bioinform Comput Biol) , 11(1): 1340004; <a href="http://www.worldscientific.com/doi/abs/10.1142/S0219720013400040">doi:10.1142/S0219720013400040</a></span></li>

  <li>(2013) <span class="title">Learning advanced TFBS models from chip-seq data diChIPMunk: Effective construction of dinucleotide positional weight matrices.</span> <span class="authors">Kulakovskiy, I.V., Levitsky, V.G., Oschepkov, D.G., Vorontsov, I.E., Makeev, V.J.</span><br/><span class="publisher">BIOINFORMATICS 2013 - Proceedings of the International Conference on Bioinformatics Models, Methods and Algorithms. <a href="http://www.scitepress.org/Portal/PublicationsDetail.aspx?ID=iMsTtW93wkk=&t=1">doi:10.5220/0004238201460150</a></span></li>

  <li>(2013) <span class="title">HOCOMOCO: a comprehensive collection of human transcription factor binding sites models.</span> <span class="authors">I.V. Kulakovskiy, Y.A. Medvedeva, U. Schaefer, A.S. Kasianov, I.E. Vorontsov, V.B. Bajic, V.J. Makeev.</span><br/><span class="publisher">Nucleic Acids Research, 41(D1): D195-D202, <a href="http://nar.oxfordjournals.org/content/41/D1/D195.full">doi:10.1093/nar/gks1089</a></span></li>
  </ol>
</details>
<br/>
My repositories: [**github**](https://github.com/VorontsovIE/), [bitbucket](https://bitbucket.org/VorontsovIE/), [lab's github](https://github.com/autosome-ru/).

Here are some toolkits and services, I've developed at work.

Databases:
* [HOCOMOCO](http://hocomoco.autosome.ru) - collection of transcription factor binding sites models. *Developed web-interface and backend. Performed substantial part of data analysis.*
* [Epifactors](http://epifactors.autosome.ru) - collection of epigenetic factors, corresponding genes and products. *Developed web-interface and backend.*

Web services:
* [Web panel](http://opera.autosome.ru) to access several command line tools developed in our lab via web interface. *Developed web-interface, contributed to development of backend.*

Toolkits:
* [PerfectosAPE](http://opera.autosome.ru/perfectosape/) - java package to analyse effect of single nucleotide variant on transcription factor binding affinity
* [MacroAPE](http://opera.autosome.ru/macroape/) - java package to perform comparison of transcription factor binding motifs (represented as PWM-s) and P-value estimation for PWM score/
* [papolarity](https://github.com/autosome-ru/papolarity) - python toolkit to analyze ribosome distribution along transcripts from Ribo-seq data.
* [pasio](https://github.com/autosome-ru/pasio) - python/cython tool to perform segmentation/denoise of coverage profile into regions of constant Poisson parameter λ (given prior distribution λ ~ Γ(α,β)). *Tool originally developed by Andrey Lando, my part was in performance optimization and packaging.*

Some of my side projects:
* [Analysis of draft dynamics](https://sysblok.ru/history/neizvestnyj-soldat/) in USSR during World War II (in Russian).
* [Analysis of sci-hub](https://habr.com/ru/post/359342/) daily activity patterns in different countries (in Russian).
* [Automatic Identification of the Repressed Families](https://ru.openlist.wiki/Шаблон:Предположительные_родственники) and further integration of collected data into wiki-based project "Open List".
* [Interactive historical atlas](http://geo-history.org) based on parsing wikipedia, created by school students under my (and my colleague) supervision.

Teaching activities:
* I'm one of the teachers and organisers of [**Slon**](https://vk.com/slon_ipm) summer school of applied programming.
* Currated several projects in bioinformatics at the [School of Molecular and Theoretical Biology](https://molbioschool.org/en/).
* Wrote part of training material for Python course in [Yandex.Lyceum](https://yandexlyceum.ru).
* Curate projects at [LearnPython](https://learn.python.ru) courses.
* Teach 10-th and 11-th grade students in Intellectual school.

Technologies:
* Languages I speak: Russian (native), English (good enough to speak and write).
* Languages I write: Ruby (expert), Python (advanced), Cython(intermediate), Java (intermediate), C++ (intermediate), Javascript (intermediate), bash (advanced).
* Other experience: Data analysis and visualization, Machine Learning, full-stack web-development (most experience with Ruby on Rails), docker, git, scraping, basic SQL, Linked Data with RDF and SPARQL, LaTeX.

Education:
Graduated Moscow Institute of Physics and Technology:
* Department of General and Applied Physics -- Institute of Theoretical and Experimental Physics (Math and Physics, bachelor's degree, 2011)
* Department of Innovation and High Technology -- Yandex School of Data Analysis (Math and Computer Science, master's degree, 2013)
* Vavilov Institute of General Genetics (Genetics, postgraduate study, 2017)

Career:
* 2012: development of marketplace for designers
* 2012 - now: Vavilov institute of General Genetics. Department of computational systems biology, research fellow

Here is my [CV](/static/Ilya_Vorontsov_CV.pdf).

Other stuff:
* My [articles](https://habr.com/ru/users/vorontsovie/posts/) for habrahabr.
* My telegram-channel about [open science](https://t.me/OpenScienceRu).

Contacts:
* telegram: <a href="https://t.me/VorontsovIE">@VorontsovIE</a>
* e-mail: <a href="mailto:vorontsov.i.e@gmail.com">vorontsov.i.e@gmail.com</a>
* phone: +7(926)616-0892
